rpi-update

```diff
@@ -296,13 +296,24 @@ function noobs_fix {
 
 }
 
+echo " *** Raspberry Pi firmware updater by Hexxeh, enhanced by AndrewS and Dom"
+echo
+echo " *** This script cannot uninstall any firmware automatically, only update it to a newer one. Use with caution."
+echo " ***  For unexperienced users it is recommended to use the pre-installed firmware package 'raspberrypi-bootloader'"
+echo " ***  or 'raberrypi-bootloader-nokernel' along with a kernel like 'linux-image-rpi-rpfv' and NOT rpi-update."
```

Comment:
Do you think an inexperienced user is going to understand what that is all about and what they should do instead. I certainly don't.